release-notes.md

```diff
@@ -0,0 +1,18 @@
+======================================================
+
+## OAK 0.2 Release Notes (June 01, 2020)
+>Compared to Oak 0.1.5 that was published on May 14, 2019 
```

Comment:
Maybe instead just add below a title for the last release with one sentence of what you remember it was about.
And remove this line from here. For example:

## v0.1.5 (May 14, 2019)
 * API changes

Reply:
OK, I will do it.